package.json

```diff
@@ -53,6 +53,7 @@
     "emojis": "^1.0.10",
     "eslint-plugin-github": "^4.3.5",
     "eslint-plugin-security": "^1.4.0",
+    "got": "^12.5.3",
```

Comment:
Why is this change here?

Reply:
got is been used in other PRs of Glee so I have also used the same here to make API calls as suggested by @Souvikns